gusto/transport_equation.py

```diff
@@ -58,8 +58,8 @@ def __init__(self, state, V, ibp="once"):
 
         # default solver options
         self.solver_parameters = {'ksp_type':'preonly',
-                                  'pc_type':'bjacobi',
-                                  'sub_pc_type': 'ilu'}
+                                  'pc_type':'lu',
+                                  'pc_factor_mat_solver_package': 'mumps'}
```

Comment:
This fixed the parallel issue #90 for the 2d_example sk_nonlinear.py. However I am not sure if this is good for accuracy etc.